examples/standardLayers/testFilt.m

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
I am not sure what this file is doing, does it need to be added?

Reply:
No - this is me trying to resolve some issues with DREAM